### PR TITLE
Implement voters list as card.

### PR DESCRIPTION
**Pull request by Alexays**
may-30-2017-1-08-am

### CODE REVIEW

**Comment by rudehn** [comment]
oh thats cool

**Comment by mark-i-m** [comment]
OOO Shiny!

File: github_api/misc.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from math import log
 
+
```
Comment (PlasmaPower) [inline github_api/misc.py]: Everyone's PR is going to include this line...

Comment (Alexays) [reply github_api/misc.py]: Oh my bad !

Comment (rudehn) [reply github_api/misc.py]: This is like number 5 or 6 with the same fix lol

**Comment by yet-another-account** [comment]
To paraphrase @mark-i-m , `Shiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiny!!!`

File: github_api/misc.py (hunk 1 above)
Comment (phil-r) [reply github_api/misc.py]: How to make everyone nervous ;)

Comment (mark-i-m) [reply github_api/misc.py]: Maybe chaosbot should not start tallying votes until travis is satisfied?

Comment (phil-r) [reply github_api/misc.py]: @mark-i-m Travis is not 100% reliable ;( there were PRs where he didn't update statuses at all

Comment (mark-i-m) [reply github_api/misc.py]: Hmm... interesting... Usually when I see that it is due to browser caching...

Comment (phil-r) [reply github_api/misc.py]: It was #344 here is the [status](https://api.github.com/repos/chaosbot/Chaos/commits/cea1616848f9182b82a0354feceecaa0e1bdc2fb/status), as you can see - it's only from chaos

Comment (mark-i-m) [reply github_api/misc.py]: Interesting

**Comment by davidak** [comment]
@Leigende Very Good! Can one scroll down and see the own position?

EDIT: I see in the code you added a link to the full list.

**Pull request merged**